### ISSUE TITLE
Question: What happens during Purge Operation?

### ISSUE THREAD

**Issue by Chewie9999** [mcguirepr89/BirdNET-Pi#1158]
Is it just the mp3 files that are deleted?
If I copied off the mp3s files before the purge happened (I have limited space on my Pi), can I somehow link them up with the detections in the database some time in the future?

**Comment by wokkeltje13** [comment]
for me "Purge" is doing nothing, the disk is just running full if I don't remove all data once in a while. Could this be an issue?

**Comment by MueJosh** [comment]
This topic fits better to "discussion", it's not an issue

> Is it just the mp3 files that are deleted?

#### Yes

> If I copied off the mp3s files before the purge happened (I have limited space on my Pi), can I somehow link them up with the detections in the database some time in the future?


The birds.db doesn't get purged, you can still see the detections (and other information)- but not the audio file. 
If you copy the audio files/folder to the intended directory (on your pi) you can see and listen to them again. 

It's also possible on your pc with SQLite, but I don't know how

**Comment by Chewie9999** [comment]
> This topic fits better to "discussion", it's not an issue

True, although it would be nice if the documentation is updated to explain this.
Thanks for the reply!

**Comment by MueJosh** [comment]
This repo doesn't get any updates for now... 

If you want a fork that's still maintained I can recommend the birdnetpi fork from @Nachtzuster

**Comment by Chewie9999** [comment]
> This repo doesn't get any updates for now...

Oh :(

> If you want a fork that's still maintained I can recommend the birdnetpi fork from @Nachtzuster

Thank you, I will check it out!

**Comment by wokkeltje13** [comment]
can you help me how to modify to a new fork? or do I need to do a fresh install?

**Comment by MueJosh** [comment]
> can you help me how to modify to a new fork? or do I need to do a fresh install?

There is no need for a fresh install 
Check out [migrating to nachtzuster fork](https://github.com/Nachtzuster/BirdNET-Pi?tab=readme-ov-file#migrating)
